setup/project_task_pull_request/setup.py

```diff
@@ -0,0 +1,6 @@
+import setuptools
+
+setuptools.setup(
+    setup_requires=['setuptools-odoo'],
+    odoo_addon=True,
+)
```

Comment:
should this file be removed from the commit?

Reply:
No, they are needed for pip